sw/host/opentitanlib/src/app/config/opentitan.json

```diff
@@ -1,9 +1,5 @@
 {
   "pins": [
-    {
-      "name": "RESET",
-      "level": true
-    },
```

Comment:
From what I can tell the "alias_of" field is necessary here for //sw/device/silicon_creator/rom/e2e/weak_straps:sw_strap_value_sim_verilator to correctly init.

Reply:
"alias_of" is needed only if the transport does not natively call the pin "RESET".  HyperDebug for instance is very versatile, and support being wired to the device under test in many ways.  We may want to have opentitan_hyperdebug.json declare that "RESET" is to be an alias of the pin that HyperDebug knows as "CN9_27" (or something, not the actual mapping).  If a transport has a pin called "RESET" already, which verilator could well have, then alias_of is nit required.

In any case, such aliases belong in the configuration files specific to the transport, not this file.

Actually in the Ti50 group, we have been debating whether it makes the more sense that the chip under test default to being held in reset, and each test can as part of its initialization set up strapping, and then explicitly release the chip from reset.